CMakeLists.txt

```diff
@@ -199,6 +193,15 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/omplConfig.cmake
     ${CMAKE_CURRENT_BINARY_DIR}/omplConfigVersion.cmake
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/ompl/cmake
     COMPONENT ompl)
+install(TARGETS ompl
+    EXPORT omplExport
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    COMPONENT ompl)
```

Comment:
Note - I have no idea what's going on with `omplapp` or `python` component, and if they should be installed here or not.